src/components/common/ClientIntakeForm/IntakeForms/DocumentUpload.js

```diff
@@ -0,0 +1,244 @@
+    label: 'A valid social security card for everyone in my household',
+    value: 'valid_social',
+  },
+  {
+    label: 'Signed up for benefits through DSHS (WIC, TANF, SNAP)',
+    value: 'dshs_wic_tanf_snap',
+  },
+  {
+    label: 'Completed the Responsible Renters Course',
+    value: 'responsible_renters_course',
+  },
+  {
+    label: 'Birth certificates for children in household',
+    value: 'birth_cert_for_children',
+  },
+  { label: 'Children Enrolled in School', value: 'child_enrolled_school' },
+  { label: 'Childcare', value: 'childcare' },
+];
+
+const documentTypeFilter = [
+  { text: 'completed_hfca', value: 'completed_hfca' },
+  { text: 'valid_driver', value: 'valid_driver' },
+  { text: 'valid_social', value: 'valid_social' },
+  { text: 'dshs_wic_tanf_snap', value: 'dshs_wic_tanf_snap' },
+  { text: 'responsible_renters_course', value: 'responsible_renters_course' },
+  { text: 'birth_cert_for_children', value: 'birth_cert_for_children' },
+  { text: 'child_enrolled_school', value: 'child_enrolled_school' },
+  { text: 'childcare', value: 'childcare' },
+];
+
+const Documents = props => {
+  const [disabled, setDisabled] = useState(true);
+  const [documentType, setDocumentType] = useState(null);
+
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      key: 'name',
+    },
+    {
+      title: 'Type',
+      dataIndex: 'documentType',
+      key: 'type',
+      filters: documentTypeFilter,
+    },
+    {
+      title: 'Options',
+      key: 'options',
+      render: (text, record, i) => (
+        <Space size="middle">
+          <Button
+            icon={<DeleteOutlined />}
+            onClick={e => handleDelete(record)}
+          ></Button>
+        </Space>
+      ),
+    },
+    {
+      title: 'Uploaded',
+      key: 'uploaded',
+      render: (text, record, i) => (
+        <Space size="middle">
+          {record.success ? (
+            <CheckCircleOutlined />
+          ) : (
+            <ExclamationCircleOutlined />
+          )}
+        </Space>
+      ),
+    },
+  ];
+
+  useEffect(() => {
+    if (props.client.id !== 0) {
+      axios
+        .get(`http://localhost:8000/api/documents/${props.client.id}`)
+        .then(documents => {
+          console.log(documents.data);
+          props.setDocuments(documents.data);
+          console.log(documents.data);
+        })
+        .catch(error => {
+          alert('ERROR ' + error);
+        });
+    }
+  }, [props.client]);
+
+  const handleDelete = file => {
+    axios
+      .post(`http://localhost:8000/api/documents/${props.client.id}`, file)
+      .then(() => {
+        props.deleteDocument(file);
+      })
+      .catch(error => {
+        alert('ERROR ' + error);
+      });
+  };
+
+  const upload = async e => {
+    console.log(e.file.name, { ...e.file });
+
+    setDisabled(true);
+    let file = e.file;
+    let fileParts = e.file.name.split('.');
+    let fileName = fileParts[0];
+    let fileType = fileParts[1];
```

Comment:
What happens if someone drags a file onto the component but the document type wasn't chosen?

Reply:
ok, I found the disabled property on the dragger. I think this interaction is complicated enough to warrant some comments on the onSelect function.

Reply:
I added a brief comment, let me know if I should further elaborate